src/library/util.cpp

```diff
@@ -861,11 +861,7 @@ void initialize_library_util() {
 
     sstream out;
 
-    out << LEAN_VERSION_MAJOR << "."
-        << LEAN_VERSION_MINOR << "." << LEAN_VERSION_PATCH;
-    if (std::strlen(LEAN_SPECIAL_VERSION_DESC) > 0) {
-        out << "-" << LEAN_SPECIAL_VERSION_DESC;
-    }
+    out << LEAN_VERSION_STRING;
```

Comment:
Maybe this is just future proofing, but as written I think `g_short_version_string` is unneeded since it is equivalent to the compile time constant `LEAN_VERSION_STRING`.